src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2009,7 +2009,7 @@
     <Field Name="OriginalIndexersOpt" Type="ImmutableArray&lt;PropertySymbol&gt;" Null="allow" SkipInNullabilityRewriter="true"/>
   </Node>
 
-  <Node Name="BoundIndexOrRangePatternIndexerAccess" Base="BoundExpression">
+  <Node Name="BoundIndexOrRangeIndexerFallbackAccess" Base="BoundExpression">
```

Comment:
>Fallback [](http://example.com/codeflow?start=38&length=8)

Same comment here. If the word is important, perhaps the comment explaining the aspect is in order.

Reply:
I think the name of the node should reflect what the node represents, not why it is created.